modules/articles/client/articles.client.module.js

```diff
@@ -1,4 +1,10 @@
-'use strict';
+(function (app) {
```

Comment:
I don't really prefer using IIFEs. It seems unnecessary and just extra for very little benefit.